library/Rules/EndsWith.php

```diff
@@ -43,7 +72,7 @@ protected function validateEquals($input)
             === mb_strlen($input, $enc) - mb_strlen($this->endValue, $enc);
     }
 
-    protected function validateIdentical($input)
+    protected function validateIdentical($input): bool
```

Comment:
```diff
--    protected function validateIdentical($input): bool
++    private function validateIdentical($input): bool
```

Since they class is final.